src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -10,6 +10,7 @@ internal static class IDEDiagnosticIds
         public const string RemoveUnnecessaryCastDiagnosticId = "IDE0004";
         public const string RemoveUnnecessaryImportsDiagnosticId = "IDE0005";
         public const string IntellisenseBuildFailedDiagnosticId = "IDE0006";
+        public const string PopulateSwitchDiagnosticId = "IDE0007";
```

Comment:
could you please choose a different Id here, ids 1 through 9 are taken, see [future branch](https://github.com/dotnet/roslyn/blob/future/src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs).